examples/tutorial/requirements.txt

```diff
@@ -1,5 +1,8 @@
 fsspec
 s3fs
-ray==1.0.0
+ray[default]>=1.4.0
```

Comment:
Can we have the tutorials rely on `modin[all]` instead of all these dependencies?